package.json

```diff
@@ -23,5 +23,8 @@
     "verbose": true,
     "testURL": "http://localhost/",
     "testEnvironment": "node"
+  },
+  "dependencies": {
+    "jszip": "^3.10.1"
```

Comment:
Your PR text said adding the package to webapp but it's installed here in root. Either functions, but I think webapp as your note said makes more sense.

Reply:
I probably literally run "npm install" without checking where I was at first 😅 thank you, that's a great call!

Reply:
I think the new `pnpm-lock.yaml` in root has to be deleted too? 🙂 I think we only use `npm` in root is why it was freshly generated.

Reply:
Done!